kyuubi-server/src/main/resources/sql/sqlite/metadata-store-schema-1.6.0.sqlite.sql

```diff
@@ -26,10 +26,10 @@ CREATE TABLE IF NOT EXISTS metadata(
     peer_instance_closed boolean default '0' -- closed by peer kyuubi instance
 );
```

Comment:
curious, as `JDBCMetadataStore#getLatestSchemaUrl` always return the latest version of metadata store schema, and the SQLite support has been available since v1.8.0, do we still need to provide `metadata-store-schema-1.6.0.sqlite.sql`?

Reply:
You are right, we don't need that. Let's fix it and remove it later, to provide a valid example for future changes.